src/components/common/FormField.tsx

```diff
@@ -85,6 +88,7 @@ const FormField: React.FC<{
   return (
     <TextField
       label={label}
+      required={required}
```

Comment:
what exactly does this do? we already have this `isValidated` prop - do we need both of these props?

Reply:
the required attribute just adds an asterisk to the label like Name*